src/api/roles/coach/coach.py

```
@@ -140,10 +142,31 @@ def update_coach_info(new_coach_details: UpdateCoachInfoInput, db = Depends(get_
+
         for e in new_coach_details.experiences:
             db.add(CoachExperience(coach_id=coach.id, experience_id=e.id)) # type: ignore
 
+    if new_coach_details.specialties is not None:
+        coach.specialties = ",".join(new_coach_details.specialties)
+
+    if new_coach_details.pricing_plan is not None:
+        pp = new_coach_details.pricing_plan
+        # Close existing open plans
+        existing_plans = db.exec(
+            select(PricingPlan).where(PricingPlan.coach_id == coach.id, PricingPlan.open_to_entry == True)
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `PricingPlan.open_to_entry:` for truth checks

Replace with `PricingPlan.open_to_entry`

(E712)